horaedb/metric_engine/src/manifest.rs

```diff
@@ -29,13 +39,16 @@ use crate::{
 
 pub const PREFIX_PATH: &str = "manifest";
 pub const SNAPSHOT_FILENAME: &str = "snapshot";
+pub const SST_PREFIX: &str = "sst";
```

Comment:
Those files should not be named sst, sst is only used in LSM, and represent sorted string table, here those file are delta files based on snapshot.